admin/data/sql/migrations/20190816105838.do.create-group-archive.sql

```diff
@@ -0,0 +1,10 @@
+create table mtc_admin.[z_group_archive]
```

Comment:
SELECT INTO  can be used for this one too.

Reply:
as above.